politeiawww/database/database.go

```diff
@@ -113,6 +113,12 @@ type User struct {
 	Deactivated                     bool      // Whether the account is deactivated or not
 	EmailNotifications              uint64    // Notify the user via emails
 
+	// Access times for proposals that have been accessed by the user. Each
```

Comment:
Do we need to store this in a map? It feels that this is light enough to go to the db every time.